src/app/session/session-state.service.ts

```diff
@@ -1,24 +1,7 @@
 import { Injectable } from '@angular/core';
 import { BehaviorSubject, Observable } from 'rxjs';
 import { map } from 'rxjs/operators';
-
-export class BudgetItem {
```

Comment:
Moved these into a separate .typings file, since sesstion-state should only contain the session-state class and constants that only it uses

Reply:
Why is it called .typings?

Reply:
.typings means is it defines types/constants